requests_auth/oauth2_tokens.py

```diff
@@ -55,34 +55,36 @@ def add_bearer_token(self, key: str, token: str):
 
         self._add_token(key, token, expiry)
 
-    def add_access_token(self, key: str, token: str, expires_in: int):
+    def add_access_token(self, key: str, token: str, expires_in: int, refresh_token: str = None):
         """
         Set the bearer token and save it
         :param key: key identifier of the token
         :param token: value
         :param expires_in: Number of seconds before token expiry
+        :param refresh_token: refresh token value
         :raise InvalidToken: In case token is invalid.
         """
         expiry = datetime.datetime.utcnow().replace(
             tzinfo=datetime.timezone.utc
         ) + datetime.timedelta(seconds=int(expires_in))
-        self._add_token(key, token, expiry.timestamp())
+        self._add_token(key, token, expiry.timestamp(), refresh_token)
 
-    def _add_token(self, key: str, token: str, expiry: float):
+    def _add_token(self, key: str, token: str, expiry: float, refresh_token: str = None):
```

Comment:
Do we actually have a call where we don't provide the refresh token (even if set to None) ?

Reply:
This method is called without a refresh token from the `add_bearer_token()` method.